test/torch/tensors/test_large_precision.py

```diff
@@ -17,23 +17,23 @@ def test_wrap(workers):
 
 def test_fix_prec(workers):
     x = torch.tensor([1.5, 2.0, 3.0])
-    enlarged = x.fix_prec(internal_type=torch.int16, precision_fractional=256)
+    enlarged = x.fix_prec(internal_type=torch.int16, precision_fractional=128)
```

Comment:
Can you give an intuition for moving 256->128?

Reply:
For a fixed_precision of 256 and this tensor, we would need to have a field of size more than 10**512 and I found it to be a bit high. But I admit that it's kind of a lightweight justification, I can change it back if needed!